app/src/main/java/com/alphawallet/app/viewmodel/HomeViewModel.java

```diff
@@ -407,8 +337,6 @@ public void handleQRCode(Activity activity, String qrCode)
                     showActionSheet(activity, qrResult);
                     break;
                 case PAYMENT:
-                    showSend(activity, qrResult);
```

Comment:
Should this be removed? That is, it may be used.

Reply:
Not removed, IDE suggests merging the 2 cases.

Reply:
> Not removed, IDE suggests merging the 2 cases.

That'll learn me to not just look at the diff.